net/netlink/netlink_conn.c

```diff
@@ -317,7 +317,7 @@ void netlink_add_broadcast(int group, FAR struct netlink_response_s *data)
 
   while ((conn = netlink_nextconn(conn)) != NULL)
     {
-      if (conn->groups & (1 << (group - 1)) == 0)
+      if (!(conn->groups & (1 << (group - 1))))
```

Comment:
We should try to be consistent with this.  I have change all of these to use (bitwise expression) == 0  instead of !(bitwise expression).  I reserve ! for use with boolean expressions.

Doesn't matter technically and there is not preference in the coding standard, but it woudl be nice if we didn't keed change each other's code only to match our personal preference.

Reply:
Done.